### PR TITLE
fix(RHINENG-5932): Fix found visual regressions after PF5 migration

### PR DESCRIPTION
**Pull request by gkarat**
Fixes some of the bugs described in https://issues.redhat.com/browse/RHINENG-5932 comments.

This:
- On the staleness page, aligns the card title and popover
- Fixes all the icons that missed colours (danger, warning)
- On the groups page, aligns horizontally the Create group button with the rest of the toolbar
- Adds missing OUIA IDs and extends the OUIA coverage for the bulk select, conditional filter and actions in PrimaryToolbar.

### CODE REVIEW

**Comment by gkarat** [comment]
I'd also bump FEC config and update OUIA IDs for conditional filters. Let's wait for https://github.com/RedHatInsights/frontend-components/pull/1972 to be merged first.

File: src/Utilities/hooks/useBulkSelectConfig.js [hunk 1]
```diff
@@ -92,7 +92,6 @@ export const useBulkSelectConfig = (
       onSelectRows(0, value);
     },
     toggleProps: {
-      'data-ouia-component-type': 'bulk-select-toggle-button',
```
Comment (leSamo) [inline src/Utilities/hooks/useBulkSelectConfig.js:95]: Is this removal intentional? What if QE tests rely on it?

**Review by AsToNlele — APPROVED** [review]
Well done @gkarat :smile:
